trackpy/feature.py

```diff
@@ -275,8 +278,7 @@ def _refine(raw_image, image, radius, coords, max_iterations,
         allow_moves = True
         for iteration in range(max_iterations):
             off_center = cm_n - radius
-            if walkthrough:
-                print_update(off_center)
+            logger.debug('off_center: %f', off_center)
             if np.all(np.abs(off_center) < GOOD_ENOUGH_THRESH):
```

Comment:
Not related to this PR but, GOOD_ENOUGH_THRESH looks like a magic number. Maybe an optional parameter to the function would be better. Same for SHIFT_THRESH.